README.md

```diff
@@ -46,13 +53,15 @@ These three libraries provide a powerful combination of HPC message passing tool
     pip install -e .
     cd ..
 
-### dask-cuda ###
+### conda-forge Dependencies ###
 
-    conda install -c rapidsai-nightly dask-cuda
+Make sure `CUDA_HOME` points to your CUDA Toolkit install.
 
-### conda-forge Dependencies ###
+    conda install -c conda-forge automake make cmake libtool pkg-config compilers nvcc_linux-64 pytest-asyncio cupy distributed
+
+### dask-cuda ###
 
-    conda install -c conda-forge automake make cmake libtool pkg-config pytest-asyncio cupy distributed
+    conda install -c rapidsai-nightly -c nvidia -c conda-forge dask-cuda
```

Comment:
@raydouglass, could you please check that this command is correct?

Reply:
Looks right to me